src/components/DevelopmentGuide/ConditionalViewport.mdx

```diff
@@ -0,0 +1,111 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Development Guide/Conditional Viewport" />
+
+# Conditional Viewport
+
+## Table of Contents
+
+- {<Link href="#background-on-the-viewport-and-input-auto-zoom-issue" target="_self">Background on the viewport and input auto-zoom issue</Link>}
+- {<Link href="#how-to-resolve" target="_self">How to resolve</Link>}
+- {<Link href="#approach-for-nextjs-app-router-apps" target="_self">Approach for Next.js App Router apps</Link>}
+- {<Link href="#approach-for-nextjs-pages-router-apps" target="_self">Approach for Next.js Pages Router apps</Link>}
+
+## Background on the viewport and input auto-zoom issue
+
+The viewport is the visible area of a web page, which varies from device to device.
+The user interacts with the viewport by scrolling or zooming in and out. The viewport's dimensions and scale can be set from the `<meta>` tag.
+
+On iOS devices, non-native websites will automatically zoom in when a user selects an input element (changing the scale of the viewport). This default is meant to ensure any input field is accessible for the visually impaired, but it generally forces users to zoom back out to continue viewing the page.
+
+This auto-zoom does not occur on Android mobile devices, so **it requires a device-specific solution to be implemented by the consuming app** (rather than, for example, changing the DS `TextInput` component to resist the zoom behavior).
```

Comment:
This is a good point. I think it works well so you don't need it in parenthesis. Maybe you can expand and say that we're building components that can be used anywhere and this issue/fix is outside the scope of the component. A component shouldn't know if it's rendering on ios, android, or any other device.